context/llpcCompiler.cpp

```diff
@@ -1616,6 +1806,7 @@ MetroHash::Hash Compiler::GenerateHashForCompileOptions(
         cl::LogFileOuts.ArgStr,
         cl::EnableShadowDescriptorTable.ArgStr,
         cl::ShadowDescTablePtrHigh.ArgStr,
+        cl::ExecutableName.ArgStr
```

Comment:
why do we need add application name in the ignore list?

Reply:
It is not strictly needed for this change, but, if I want to prime the cache using one application, say `amdllpc`, and then use it in the actual game, the executable name has to be ignored.  This is important for us.  It is here because this is part of my internal testing for the feature.

Secondly, does the name effect the generated code?  If not, then there is no reason for the cache to be rejected.  If there is a reason for the exe name to be included then I can remove this change for the PR, and we can have a separate discussion on how to handle it.

Reply:
1. this hash only affect the file name of internal shader cache. it doesn't affect either the shader key of compile results or the compile result.
2. currently, amdllpc only integrate the internal shader cache, it doesn't support pipleine binary cache, which is used as vulkan application binary cache. do you plan to do it by amdllpc or other tools?

Reply:
I intend on using amdllpc with appropriate options as the cache creator.